pallet-logion-loc/src/lib.rs

```diff
@@ -578,7 +582,24 @@ pub mod pallet {
 
         #[cfg(feature = "try-runtime")]
         fn post_upgrade(_state: Vec<u8>) -> Result<(), sp_runtime::DispatchError> {
+
             assert_eq!(PalletStorageVersion::<T>::get(), StorageVersion::default());
+
+            LocMap::<T>::iter().for_each(|entry| {
+                let loc_id = entry.0;
+                let loc = entry.1;
+                loc.metadata.iter().for_each(|metadata| {
+                    log::info!("LOC {:?} metadata {:?} acknowledged_by_owner {:?} acknowledged_by_verified_issuer {:?}", loc_id, metadata.name, metadata.acknowledged_by_owner, metadata.acknowledged_by_verified_issuer);
```

Comment:
In the future, I think we should keep logging only when assertions cannot be used.